packages/types/src/global-settings.ts

```diff
@@ -230,6 +231,7 @@ export const SECRET_STATE_KEYS = [
 // Global secrets that are part of GlobalSettings (not ProviderSettings)
 export const GLOBAL_SECRET_KEYS = [
 	"openRouterImageApiKey", // For image generation
+	"kiloCodeImageApiKey",
```

Comment:
```suggestion
	"kiloCodeImageApiKey", // kilocode_change
```

Reply:
it already says kilocode, so I thought it wasn't super necessary